include/z64save.h

```diff
@@ -910,6 +913,7 @@ typedef enum {
 #define WEEKEVENTREG_37_01 PACK_WEEKEVENTREG_FLAG(37, 0x01)
 #define WEEKEVENTREG_37_02 PACK_WEEKEVENTREG_FLAG(37, 0x02)
 #define WEEKEVENTREG_37_04 PACK_WEEKEVENTREG_FLAG(37, 0x04)
+
```

Comment:
Any reason this extra newline was added?

Reply:
I think the reason I put it in was to separate the group of flags used for EnSGoro. I can remove the newline for now and put it back in for a different PR related to those events.

Reply:
Ahhhh, I see. I thought it was just accidental; I've certainly inserted accidental newlines in the past. If it's an intentional chane, that's fine!